src/magick-image.ts

```diff
@@ -938,6 +938,27 @@ export interface IMagickImage extends IDisposable {
      */
     flop(): void;
 
+    /**
+     * Gaussian blur image.
+     * @param radius - The number of neighbor pixels to be included in the convolution.
+     */
+    gaussianBlur(radius: number): void;
```

Comment:
The Magick.NET library also has a `(radius: number, channels: Channels)` overload. Can this also be added?

Reply:
For an other method I made the `channels` argument the first argument but that is also not a really clean solution that I should probably revert.